internal/controller/git/git.go

```diff
@@ -165,6 +165,7 @@ type repo struct {
 	dir                   string
 	currentBranch         string
 	insecureSkipTLSVerify bool
+	useSSH                bool
```

Comment:
```suggestion
```

We don't need this because it's safe to unconditionally set the `GIT_SSH_COMMAND` env var.